### DISCUSSION TITLE
Add Rate Limit, already supported by Borg

### DISCUSSION THREAD

**Discussion by m3nu** [borgbase/vorta#817] [Ideas]
Offer an option to set a rate limit via [`--remote-ratelimit`](https://borgbackup.readthedocs.io/en/stable/faq.html#is-there-a-way-to-limit-bandwidth-with-project-name)

Suggested by Boris via email.

**Comment by samuel-w** [comment]
This could be implemented as a checkbox under the Repository tab. 
It would require another schema update though, and we currently have many pull requests that update the SCHEMA_VERSION to 15. What should we do about this?
